packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -0,0 +1,193 @@
+                                        key={index}
+                                        disabled={!!selectionSize}
+                                        {...rest}
+                                        onClick={(e) => {
+                                            onClick?.(e);
+                                            handleClose();
+                                        }}
+                                    >
+                                        {!!icon && <ListItemIcon sx={{ minWidth: "unset !important" }}>{icon}</ListItemIcon>}
+                                        <ListItemText primary={label} />
+                                    </MoreActionsMenuItem>
+                                    {!!divider && <CrudMoreActionsDivider {...dividerProps} />}
+                                </>
+                            );
+                        })}
+                    </CrudMoreActionsGroup>
+                )}
+
+                {!!overallActions?.length && !!selectiveActions?.length && <CrudMoreActionsDivider {...dividerProps} />}
+
+                {!!selectiveActions?.length && (
+                    <CrudMoreActionsGroup
+                        groupTitle={<FormattedMessage id="comet.dam.moreActions.selectiveActions" defaultMessage="Selective actions" />}
+                        {...groupProps}
+                    >
+                        {selectiveActions.map((item, index) => {
+                            if (!item) return;
+
+                            const { divider, label, icon, onClick, ...rest } = item;
+
+                            return (
+                                <>
+                                    <MoreActionsMenuItem
+                                        key={index}
+                                        disabled={!selectionSize}
+                                        {...rest}
+                                        onClick={(e) => {
+                                            onClick?.(e);
+                                            handleClose();
+                                        }}
+                                    >
+                                        {!!icon && <ListItemIcon sx={{ minWidth: "unset !important" }}>{icon}</ListItemIcon>}
+                                        <ListItemText primary={label} />
+                                        {!!selectionSize && (
+                                            <MoreActionsSelectedItemsChip size="small" color="primary" {...chipProps} label={selectionSize} />
+                                        )}
+                                    </MoreActionsMenuItem>
+                                    {!!divider && <CrudMoreActionsDivider {...dividerProps} />}
+                                </>
+                            );
+                        })}
+                    </CrudMoreActionsGroup>
+                )}
+            </Menu>
+        </>
+    );
+}
```

Comment:
Please add the types to the theme to allow overriding the styles and default props: https://docs.comet-dxp.com/docs/comet-core-development/creating-customizable-admin-components/#adding-the-component-to-muis-theme-type

Reply:
Ah yes I forgot that yesterday. Thanks for reminding me!